x/cron/keeper/keeper.go

```diff
@@ -205,8 +210,9 @@ func (k *Keeper) executeSchedule(ctx sdk.Context, schedule types.Schedule) error
 			)
 			return err
```

Comment:
in case of error, you are missing both `telemetry.ModuleMeasureSince` calls, it's better  to use `defer`